### ISSUE TITLE
FID score: nan

### ISSUE THREAD

**Issue by Jamie-Cheung** [asarigun/TransGAN#12]
Thank you for your contribution. But in the training processing, FID score is Nan. I want to known whether it is appropriate. Should I make some chance to solve this problem?

**Comment by asarigun** [comment]
Hi @Jamie-Cheung, sorry for the late answer. Thank you so much for being interesting in the repo! Did you change the loss function in the training process or try the hyperparameters that I used in the ```train.py```? During the training, I also got some ```Nan``` values, but when I went on the training, the FID score continued to converge. Also, another issue #9 was confronted during the training. It might be helpful for the problem.

Best,

Ahmet
